Sources/TuistGraph/Models/Workspace.swift

```diff
@@ -47,7 +56,7 @@ public struct Workspace: Equatable, Codable {
     public var ideTemplateMacros: IDETemplateMacros?
     public var additionalFiles: [FileElement]
     public var lastUpgradeCheck: Version?
-    public var generationOptions: GenerationOptions?
+    public var generationOptions: [GenerationOption]
```

Comment:
As with the public API, I think a `struct` is more suitable for those options. I would propose discussing it separately and decide if it should change to an enum or ProjectOption should change to a struct for consistency